src/objc.ts

```diff
@@ -370,6 +370,7 @@ export interface Class {
   baseClassName:string;
   classMethods:Method[];
   comments:Comment[];
+  generics:string[];
```

Comment:
Sounds good, I like the name `covariantTypes`. Am also totally down with using a Maybe type, wasn't sure what the preferred way was here. Will fix.